adapters/nextmillennium/params_test.go

```diff
@@ -0,0 +1,46 @@
+package nextmillennium
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, validParam := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderNextMillennium, json.RawMessage(validParam)); err != nil {
+			t.Errorf("Schema rejected NextMillennium params: %s", validParam)
+		}
+	}
+}
+
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json-schemas. %v", err)
+	}
+
+	for _, invalidParam := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderNextMillennium, json.RawMessage(invalidParam)); err == nil {
+			t.Errorf("Schema allowed unexpected params: %s", invalidParam)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"placement_id": "123654"}`,
+}
+
+var invalidParams = []string{
+	`{"placement_id": 123654}`,
+	`nil`,
+	``,
+	`[]`,
+	`true`,
+}
```

Comment:
Please add an invalid params test for an empty object `{}`.

Reply:
@SyntaxNode Done!